routes/article.js

```diff
@@ -0,0 +1,99 @@
+import express from 'express';
+import {
+  validateId,
+  validateCommentId,
+  validateArticleData,
+  validateCommentData,
+  validateArticleUpdateData,
+} from './validation.js';
+import { articleService, commentService } from './services.js';
+
+const router = express.Router();
+
+// 게시글 목록 조회 API
+router
+  .route('/')
+  .get(async (req, res) => {
+    const { articles, pagination } = await articleService.getAllArticles(
+      req.query
+    );
+    res.send({ articles, pagination });
+  })
+  // 게시글 작성 API
+  .post(validateArticleData, async (req, res) => {
+    const { title, content, author } = req.body;
+    const newArticle = await articleService.createArticle(
+      title,
+      content,
+      author
+    );
+    res.status(201).json(newArticle);
+  });
+
+// 게시글 상세 조회 API
+router
+  .route('/:id')
+  .get(validateId, async (req, res) => {
+    const { id } = req.params;
+    const article = await articleService.getArticleById(id);
+    if (!article) {
+      return res.status(400).json({ message: '존재하지 않는 게시글입니다.' });
+    }
+    res.status(200).json(article);
+  })
+  // 게시글 수정 API
+  .patch(validateId, validateArticleUpdateData, async (req, res) => {
+    const { id } = req.params;
+    const { updateData } = req.body;
+    const updateArticle = await articleService.updateArticle(id, updateData);
+    res.status(200).json(updateArticle);
+  })
+  // 게시글 삭제 API
+  .delete(validateId, async (req, res) => {
+    const { id } = req.params;
+    await prisma.article.delete(id);
```

Comment:
prisma를 직접 사용하고계신데, 이 코드 파일에서 prisma는 사용할 수 없습니다.
구현해놓으신 service의 delete함수를 호출하도록 변경하셔야겠네요.
```suggestion
    await articleService.deleteArticle(id);
```